code/index.html

```diff
@@ -4,30 +4,44 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
     <title>Chat Bot</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
-    <main>
-      <section class="chat" id="chat"></section>
+    <div class="video-container">
+      <video class="video" autoplay muted loop playsinline>
+        <source src="video.mp4" type="video/mp4">
+        Your browser does not support the video tag
+      </video>
+      <button type="button" class="video-btn">
+        <span>Pause</span>
+        <span>Play</span>
+        <span class="toggle"></span>
 
-      <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <input id="name-input" type="text" />
-          <button class="send-btn" type="submit">
-            Send
-          </button>
-        </form>
-      </div>
-    </main>
+      </button>
+    </div>
+    <div class="chatbot">
+      <h1>Dr. Strange 👨🏻‍⚕️ </h1>
+      <main >
+        <section class="chat" id="chat"></section>
+  
+        <div class="input-wrapper" id="input-wrapper">
+          <form class="" id="name-form">
```

Comment:
We noticed that the class for the form is empty, is it on purpose or is there a typo maybe? :)